README.md

```diff
@@ -400,6 +400,7 @@ function onInputKeyDown(event) {
 | loadingPlaceholder | string\|node | 'Loading...' | label to prompt for loading search result |
 | tabSelectsValue | bool | true | whether to select the currently focused value when the `[tab]` key is pressed |
 | value | any | undefined | initial field value |
+| valueComponent | func | undefined | function that returns a user-created component to render in the value field `() => <CustomComponent ... />` |
```

Comment:
The default value for a `valueComponent` is a Value component and not undefined. :smile:

This is a nit but to match the wording/format of the other documentation can you update it to say
function which returns a custom way to render/manage the value selected. `<CustomValue />`